skimage/util/montage.py

```diff
@@ -108,3 +121,68 @@ def montage2d(arr_in, fill='mean', rescale_intensity=False, grid_shape=None):
     arr_out = arr_out.reshape(alpha_y * height, alpha_x * width)
 
     return arr_out
+
+
+def montage_rgb(arr_in, fill='mean', grid_shape=None, padding_width=0):
+    """Create a 3-dimensional 'montage' from a 4-dimensional input array
+    representing an ensemble of equally shaped 3-dimensional images.
+
+
+    Parameters
+    ----------
+    arr_in : ndarray, shape=[n_images, height, width, n_channels]
```

Comment:
Should use standard notation here: `(K, M, N[, C])`, and move the detailed description to the line below.